class/defaults.yml

```diff
@@ -22,6 +22,7 @@ parameters:
       - 'imagestreamimports'
       - 'imagestreammappings'
       - 'mutations'
+      - 'podsecuritypolicysubjectreviews'
```

Comment:
`'.+reviews'` is already known to fail further up. Bug in the script?

Reply:
https://github.com/projectsyn/k8s-object-dumper/blob/master/dump-objects#L223 seems legit.